lib/metadata/util/win32/Win32System.rb

```diff
@@ -163,7 +163,7 @@ def initialize(_c, fs)
       # @debug_str = ''
 
       # Force memory cleanup
-      software_doc = nil; sys_doc = nil; GC.start
```

Comment:
Don't we want to set these to nil so GC will clean them up?

Reply:
I see.  yes - CodeClimate (and I) didn't understand what was happening.  I will undo.